meshparty/trimesh_io.py

```diff
@@ -362,10 +363,11 @@ def mesh(self, filename=None, seg_id=None, cache_mesh=True,
         return mesh
 
 class Mesh(trimesh.Trimesh):
-    def __init__(self, *args, mesh_edges=None, **kwargs):
+    def __init__(self, *args, non_face_edges=None, **kwargs):
```

Comment:
This change leads to inconsistencies within this same file!
Most notable: https://github.com/sdorkenw/MeshParty/blob/2ff7a0ab223d4e8816d941b43d2e7f383da6063d/meshparty/trimesh_io.py#L173
https://github.com/sdorkenw/MeshParty/blob/2ff7a0ab223d4e8816d941b43d2e7f383da6063d/meshparty/trimesh_io.py#L305
https://github.com/sdorkenw/MeshParty/blob/2ff7a0ab223d4e8816d941b43d2e7f383da6063d/meshparty/trimesh_io.py#L359

And there are some other namings of `mesh_edges`...

Reply:
i believe i have addressed these now.

Reply:
There is still this here left:
https://github.com/sdorkenw/MeshParty/blob/vtk_skeleton_refactor/meshparty/trimesh_io.py#L738

I believe we should deprecate this function and default to the new merging

Reply:
fixed the unused thing.. agreed about deprecating merge_large_components.